lib/src/spotify_browser.dart

```diff
@@ -3,47 +3,15 @@
 
 part of spotify;
 
-class SpotifyApi extends SpotifyApiBase {
-  SpotifyApi(SpotifyApiCredentials credentials) : super(credentials);
 
-  @override
-  Future<String> _getImpl(String url, Map<String, String> headers) async {
```

Comment:
Overall I like having a more dynamic approach - better not having this "duplicate" code and just passing in a client.